spec/shore/client_spec.rb

```diff
@@ -0,0 +1,39 @@
+RSpec.describe Shore::Client do
+  describe '.with_access_token' do
+    before do
+      stub_request(:get,
+                   "#{Shore::Client::Messaging.base_uri(:v1)}/messages/abc1")
```

Comment:
Instead of `abc1` use a valid uuid, just to make the test a little more realistic.